src/contracts/periphery/OperatorFactory.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.0;
+
+import "./Operator.sol";
+import "@openzeppelin/contracts/utils/Create2.sol";
+
+contract OperatorFactory {
```

Comment:
You might want to consider having every instance be a minimal proxy to reduce deploy costs. Tradeoff is the additional `delegatecall` for every call (May or may not be worth it). If deploys are decently cheap and a one time expense then we can keep as is.

Reply:
True, figured we don't want the additional `delegateCall`